sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -401,9 +400,13 @@ def enable_tracing=(enable_tracing)
       @traces_sample_rate ||= 1.0 if enable_tracing
     end
 
+    def traces_sample_rate=(traces_sample_rate)
```

Comment:
Let's raise `ArgumentError` here and in `profiles_sample_rate=` if the argument isn't a `Numeric`.

We already raise `ArgumentError` when users pass non-callable values to callable configs. So this aligns with the current practice.

Reply:
@st0012 I've implemented this, but it was also necessary to allow `nil` since that is the default value for both sample rates. Otherwise, this causes many tests to fail.

Reply:
Yeah we need to take `nil` too.